src/tsconfig/rulesets/generated.ts

```diff
@@ -0,0 +1,32 @@
+import { BASE_COMPILER_OPTIONS, convertForJson } from '../compiler-options';
+import { Match, RuleSet, RuleType } from '../validator';
+
+// legacy settings configurable via jsii
+// we accept all value for these
+export const configurableViaJsiiSettings = new RuleSet();
+configurableViaJsiiSettings.pass('outdir', Match.ANY);
```

Comment:
`passMany` ?

Perhaps it would even be nicer if `RuleSet` accepted the `passMany` dictionary as a constructor argument.

Reply:
I got rid of passMany/failMany. It was a bit weird that it was responsible for the conversion from value to Matcher. We might want to re-introduce this later as something that takes `Record<string, Matcher>` to make it easier to define many rules. But for now this will do.